beps/bep_0048.rst

```diff
@@ -0,0 +1,90 @@
+:BEP: 48
+:Title: Tracker Protocol Extension: Scrape
+:Version: $Revision$
+:Last-Modified: $Date$
+:Author:  Jimmy Zelinskie <jimmyzelinskie@gmail.com>
+:Status:  Draft
+:Type:    Standards Track
+:Content-Type: text/x-rst
+:Created: 08-Sep-2016
+:Post-History:
+
+
+Abstract
+--------
+
+The original BitTorrent Protocol Specification (BEP 003) defines an exchange between a client and a tracker referred to as an Announce [#BEP_003]_.
+An Announce has a two objectives: update a peer's presence in the Swarm and discover other peers in the Swarm.
+As a result of dealing with multiple objectives, this exchange is computationally expensive when exercised at scale.
+In order to prevent trackers from becoming overloaded, announces are specified to be sent on an interval dictated by the tracker's Announce response or when a client event occurs.
+However, in order to build responsive user interfaces, clients desired a way to query metadata about Swarms in bulk in the least computationally expensive way.
+The exchange that fetches this metadata for the clients is referred to as a Scrape.
+
+
+Implementation
+--------------
+
+As the UDP Tracker Protocol Specification (BEP 015) already defines the implementation of Scrape using the UDP protocol [#BEP_015]_, the following will document the implementation for the HTTP Protocol.
+
+Changes to Announce
+...................
+
+The URL in which the Announce endpoint lives is now expected to terminate with ``/announce``.
+This has been modified so that clients can rewrite this section to ``/scrape`` to perform Scrape requests.
+The standard URL query parameters defined in BEP 003 still follow [#BEP_003]_.
+There are no changes to Announce responses.
+
+Scrape Request
+..............
+
+As per the previous section, clients should rewrite the final ``/announce`` portion of the Announce URL provided via a metadata file or magnet link to ``/scrape``.
```

Comment:
I've read that some clients might rewrite `announce.php` into `scrape.php` as well. I'm not sure how this type of behavior should be specified. An announce is required to have the word "announce" on the URL path that should be able to be programmatically rewritten into the word "scrape"?